Library/Formula/dockviz.rb

```diff
@@ -0,0 +1,19 @@
+class Dockviz < Formula
+  desc "Visualizing docker data"
+  homepage "https://github.com/justone/dockviz"
+  url "https://github.com/justone/dockviz.git", :tag => "v0.3", :revision => "ae855d6a5a540c3563948e7035f9d19cab17f5b0"
+  head "https://github.com/justone/dockviz.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    system "go", "get", "github.com/fsouza/go-dockerclient"
```

Comment:
Please use `go_resource`s rather than `go get`, Thanks. We pretty heavily frown on remote fetching happening once we're in the `def install` execution process.

There's a bunch of examples available by searching for `go_resource` if [this one](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/consul-template.rb) doesn't fit your purpose :)